client/my-sites/stats/features/modules/shared/stats-empty-action-email.tsx

```diff
@@ -13,14 +13,20 @@ import { isJetpackSite } from 'calypso/state/sites/selectors';
 import { getSelectedSiteId } from 'calypso/state/ui/selectors';
 import type { StatsEmptyActionProps } from './';
 
-const HELP_CENTER_STORE = HelpCenter.register();
-
 const StatsEmptyActionEmail: React.FC< StatsEmptyActionProps > = ( { from } ) => {
 	const translate = useTranslate();
 	const siteId = useSelector( getSelectedSiteId );
-	const isJetpack = useSelector( ( state ) => isJetpackSite( state as any, siteId ) );
+	const isJetpack = useSelector( ( state ) =>
+		isJetpackSite( state, siteId, { treatAtomicAsJetpackSite: false } )
+	);
+
+	const supportLink = localizeUrl(
+		isJetpack ? JETPACK_SUPPORT_NEWSLETTER_URL : NEWSLETTER_SUPPORT_URL
+	);
 
-	const { setShowSupportDoc } = useDataStoreDispatch( HELP_CENTER_STORE );
+	const { openSupportDoc } = useSupportDocData( {
+		supportLink,
+	} );
 
 	return (
 		<EmptyStateAction
```

Comment:
Where can I test this empty state action? I don't see in any screen the "Send emails with Newsletter" text.

Reply:
Sorry about that! I've updated the description to contain the test steps for it.